vulkaninfo/vulkaninfo.c

```diff
@@ -1935,15 +2009,13 @@ static void AppDevDump(const struct AppGpu *gpu, FILE *out) {
     }
 
     bool first_in_list = true;   // Used for commas in json output
-    for (VkFormat fmt = 0; fmt < VK_FORMAT_RANGE_SIZE; ++fmt) {
-        AppDevDumpFormatProps(gpu, fmt, &first_in_list, out);
-    }
-
-    for (VkFormat fmt = VK_FORMAT_G8B8G8R8_422_UNORM; fmt <= VK_FORMAT_G16_B16_R16_3PLANE_444_UNORM; ++fmt) {
-        AppDevDumpFormatProps(gpu, fmt, &first_in_list, out);
-    }
-    for (VkFormat fmt = VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG; fmt <= VK_FORMAT_PVRTC2_4BPP_SRGB_BLOCK_IMG; ++fmt) {
-        AppDevDumpFormatProps(gpu, fmt, &first_in_list, out);
+    for (unsigned int i = 0; i < sizeof(supported_format_ranges)/sizeof(supported_format_ranges[0]); i++) {
```

Comment:
A bit of a nit pick.  You might want to change the `unsigned int` here to a `uint32_t` for consistency with the rest of the file.

Reply:
I love nit picks - it shows the reviewer is being thorough. :-)  Thanks for the review...